staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -700,7 +704,6 @@ func (f *featureGate) DeepCopy() MutableVersionedFeatureGate {
 	fg.known.Store(known)
 	fg.enabled.Store(enabled)
 	fg.enabledRaw.Store(enabledRaw)
-	fg.queriedFeatures.Store(map[Feature]struct{}{})
```

Comment:
queriedFeatures needs to be reset to a new set for a new feature gate.

Reply:
fixed